msiafterburner/tools/chocolateyInstall.ps1

```diff
@@ -27,3 +29,21 @@ Install-ChocolateyInstallPackage @packageArgs
 
 Remove-Item $unpackFile -Recurse -Force
 Remove-Item $file -Recurse -Force
+
+function InstallShortcut {
+  param (
+    $ShortcutPath
+  )
+
+  $shortcutArgs = @{
+    shortcutFilePath = $ShortcutPath
+    workingDirectory = "${env:ProgramFiles(x86)}\RivaTuner Statistics Server"
+    targetPath       = "${env:ProgramFiles(x86)}\RivaTuner Statistics Server\RTSS.exe"
```

Comment:
This is probably fine, but I'm wondering if there's some Chocolatey functionality that could be leveraged to make this more resilient to the install location being changed in the future. I think Chocolatey makes shim .exe's - would it make sense to point at one of those? I really don't know.

Reply:
https://github.com/chocolatey-community/chocolatey-packages/blob/master/extensions/chocolatey-core.extension/extensions/Get-AppInstallLocation.ps1

This function seems to work.